var/spack/repos/builtin/packages/py-libsonata/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyLibsonata(PythonPackage):
+    """
+    `libsonata` provides C++ / Python API for reading SONATA Nodes / Edges
+
+    See also:
+    https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
+    """
+    homepage = "https://github.com/BlueBrain/libsonata"
+    git = "https://github.com/BlueBrain/libsonata.git"
+
+    version('develop', git=git, submodules=True)
+
+    variant('mpi', default=False, description="Enable MPI backend")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('libsonata@develop:', type='build')
```

Comment:
If we have an `mpi` variant, shouldn't this be build with/without `mpi`? @pramodk ?